### PR TITLE
Finish Post and Comment models

### PR DESCRIPTION
**Pull request by CameronHamidi**
## Overview

Create models for Posts and Comments, to be used for creating/responding to requests for casual sports games

## Changes Made

Created two structs, `Post` and `Comment`.

## Test Coverage

I included the same fields that the backend uses, but extensive testing is not possible until the routes are finishes.

## Next Steps (delete if not applicable)

These models will likely be changed once routes and designs are finished.

## Related PRs or Issues (delete if not applicable)

https://github.com/cuappdev/uplift-backend/pull/81

### CODE REVIEW

File: Uplift/Models/Comment.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import Foundation
+
+struct Comment: Codable {
+
+    let createdAt: Date!
+    let id: Int!
+    let postId: Int!
+    let text: String!
+    let userId: Int!
```
Comment (kevinchan159) [inline Uplift/Models/Comment.swift]: remove `!` from all

File: Uplift/Models/Post.swift [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import Foundation
+
+struct Post: Codable {
+
+    let comment: [Comment]!
+    let createdAt: Date!
+    let id: Int!
+    let text: String!
+    let userId: Int!
```
Comment (kevinchan159) [inline Uplift/Models/Post.swift]: same here

File: Uplift/Views/Cells/LoadingItemCollectionViewCell.swift [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import SnapKit
+
+class LoadingSmallGridCollectionViewCell: UICollectionViewCell {
+    
+    private var collectionView: UICollectionView!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+
```
Comment (kevinchan159) [inline Uplift/Views/Cells/LoadingItemCollectionViewCell.swift]: remove new lines

File: Uplift/Views/Cells/LoadingItemCollectionViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,26 @@
+//
+//  LoadingSmallGridCollectionViewCell.swift
+//  Uplift
+//
+//  Created by Cameron Hamidi on 2/7/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
```
Comment (kevinchan159) [inline Uplift/Views/Cells/LoadingItemCollectionViewCell.swift]: alphabetize

File: Uplift/Views/LoadingScreenHomeView.swift [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+import SnapKit
+
+class LoadingScreenHomeView: UIView {
+
+    let navBarGreyedOut = UIView()
+    let tabBarGreyedOut = UIView()
+    
+    
+
+}
```
Comment (kevinchan159) [inline Uplift/Views/LoadingScreenHomeView.swift]: I feel like we should just put these UI files in the next PR when you actually put stuff inside

**Comment by CameronHamidi** [comment]
@kevinchan159 The loading screen code is still wip, no idea how it got onto this branch but I'll remove it. Was planning on using `SkeletonView` with it instead

**Pull request merged**